arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -3107,45 +3115,52 @@ void RestReplicationHandler::handleCommandRevisionRanges() {
   {
     VPackObjectBuilder obj(&response);
     TRI_voc_rid_t resumeNext = ctx.resume;
-    VPackSlice range = body.at(current);
+    VPackSlice range;
+    std::size_t left;
```

Comment:
Ii suggest to use `uint64_t` throughout for revs. We even have `TRI_voc_rid_t` which is `uint64_t`, even on 32bit machines! `size_t` on 32bit machines will be 32bit only, so this will probably not even compile without warnings on Windows at least. Let's change things here to `uint64_t` and then do another PR to change `size_t` in lots of places (including the MerkleTree files).